board/app/helpers/html_helper.php

```diff
@@ -10,4 +10,62 @@ function redirect($url)
 	header("Location: " . $url);
 	exit();
 }
-?>
+
+function time_difference($date)
```

Comment:
There are a lot of things that are being processed in this 1 function:
1. getting the current_date/time
2. getting the difference between the current_date/time vs. a different_date/time
3. putting a different label if the difference from current_date/time is greater than 1 day ago or not to display either "yesterday" or "days ago"
4. computation of the difference of values of the 'day' and 'time' between the current and previous timeframe

Plus there are magic numbers. :sweat_smile: 

You have to breakdown the codes into separate functions so that they're reusable to any part of the codes.
Example, what if we add a function for just getting the current date/time? Then you can simply just call that function anywhere in your codes.

You can simply call a function, inside another function if needed, so better to breakdown the responsibilities of the codes into different functions. (Review Art of Readable Code: "Breaking Down Giant Expressions p. 89 :dart:  )